presto-spi/src/main/java/com/facebook/presto/spi/plan/DeleteNode.java

```diff
@@ -29,7 +29,7 @@
 import static java.util.Objects.requireNonNull;
 
 @Immutable
-public class DeleteNode
+public final class DeleteNode
```

Comment:
To pass this test https://github.com/prestodb/presto/blob/0ee4687d038c2f4dd8b513a1375a39ffd8bda1eb/presto-main/src/test/java/com/facebook/presto/sql/planner/optimizations/TestConnectorOptimization.java#L90